Bottom: ed212559043bacac232c481a125c8cfff34c5b04
Top:    325288fb57f74c4e563982c6fce059e5f05de9c8
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-04-06 00:05:58 +0100

Refresh of gtk-hid-use-clipping-region-wh

---

diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 1ed9082..7a46672 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -532,6 +532,9 @@ void ghid_invalidate_all ();
 void ghid_get_coords (const char *msg, int *x, int *y);
 gint PCBChanged (int argc, char **argv, int x, int y);
 
+
+
+
 extern GdkPixmap *XC_hand_source, *XC_hand_mask;
 extern GdkPixmap *XC_lock_source, *XC_lock_mask;
 extern GdkPixmap *XC_clock_source, *XC_clock_mask;
